src/structures/ShardManager.ts

```diff
@@ -0,0 +1,19 @@
+import DiscordWebsocket from '../../ws/DiscordWebsocket';
+import Collection from '../util/Collection';
+
+export default class ShardManager {
+  shards: Collection<DiscordWebsocket>;
+
+  private shardID: number;
+
+  constructor() {
+    this.shards = new Collection({ base: DiscordWebsocket });
+    this.shardID = 0;
```

Comment:
This won't work if we need to remove shards. Also it will be easily confused with the shard manager containing a shard ID